.changelog/1338.bugfix.md

```diff
@@ -0,0 +1 @@
+Fix RPC native balance
```

Comment:
And adds PontusX RPC (supersedes https://github.com/oasisprotocol/explorer/pull/1318)